source/content/guides/build-tools/02-create-project.md

```diff
@@ -1,42 +1,51 @@
 editpath: build-tools/02-create-project.md
 image: buildToolsGuide-thumb
 ---
-In this section we'll use the Terminus Build Tools Plugin to create a new Pantheon Site, a corresponding GitHub repository, and configure CircleCI to run tests.
 
-1. Create a new project (replace `pantheon-d8-composer-project` with the name of your new site):
+In this section, we will use the Terminus Build Tools Plugin to create a new project consisting of a *Git Provider* repository, *CI Provider*, and a Pantheon site.
 
-  ```bash
-  terminus build:project:create d8 pantheon-d8-composer-project
-  ```
+## Prerequisites
 
-  <Alert title="Note" type="info">
+Before beginning, make sure you have the latest versions of Terminus and the Terminus Build Tools plugin installed. If you need help installing either, expand the installation instructions below.
 
-  Pantheon also maintains Composer based examples for [WordPress](https://github.com/pantheon-systems/example-wordpress-composer) and [Drupal 7](https://github.com/pantheon-systems/example-drops-7-composer) that are currently in alpha, requiring `--stability=alpha` in the command line options. While this guide demonstrates Drupal 8, the same workflow can be achieved on all frameworks.
+1. Install <a href="https://getcomposer.org">Composer</a>. You can verify your installation by running `composer --version`. 
 
-  </Alert>
+2. Install the most recent release of <a href="/docs/terminus/">Terminus</a>. You can verify installation and your current version by running `terminus --version`. 
+
+3. <a href="/docs/ssh-keys/">Add an SSH key</a> within your User Dashboard to enable passwordless access and avoid authentication prompts. Otherwise, provide your Pantheon Dashboard credentials when prompted.
+
+4. <a href="https://dashboard.pantheon.io/machine-token/create">Generate a Pantheon machine token</a>, then authenticate Terminus.
+
+Create the `$HOME/.terminus/plugin` directory if it does not already exist. 
+
+5. Install the <a href="https://github.com/pantheon-systems/terminus-build-tools-plugin">Terminus Build Tools Plugin</a>
+
+```
 
-  Provide additional information as prompted, such as Organization (if any), and tokens for GitHub and CircleCI access:
+mkdir -p ~/.terminus/plugins
+composer create-project --no-dev -d ~/.terminus/plugins pantheon-systems/terminus-build-tools-plugin:^2.0.0-beta17
 
-  ![Create Project Prompts](../../../images/pr-workflow/build-env-create-project-prompts.png)
+```
 
-  This process will create a secure keypair, with the public key going to Pantheon and the private key stored in CircleCI. If you remove either key, you will need to [generate a new pair](/ssh-keys) manually to fix the build process.
+## Create a Build Tools Project
+
+1. Create a new project using the command below and passing in your *Git Provider* of choice (github, bitbucket, gitlab)
+
+Starting a new Github Project
+```
+terminus build:project:create --git={yourGitProviderHere} --team='My Agency Name' wp my-site
```

Comment:
If this is specific to starting a new Github project, should we be specific about this here?

Reply:
I was trying to keep it neutral because I'm not sure how to structure around the three options yet (hopefully that's something that docs can help with) - was trying to remove references to github, oops!

Reply:
Writing it out as:

- Starting a Github project
`code`
- Starting a Bitbucket project
`code`
- Starting a Gitlab project
`code`

started to get kind of long and hard to read through so I wasn't sure if it was better to stay neutral or explicitly pick a new option (and make a note of it) or have separate installation pages for each?

Reply:
Selected Github for now and put a note that you can swap the provider + service option.